Contest_API.md

```diff
@@ -759,11 +759,11 @@ Properties of contest objects:
 | logo                         | array of FILE ? | Logo for this contest, intended to be an image with aspect ratio near 1:1. Only allowed mime types are image/\*.
 | location                     | LOCATION ?      | Location where the contest is held.
 
-The expected/typical use of `countdown_pause_time` is that once a
-`start_time` is defined and close, the countdown may be paused due to
-unforeseen delays. In this case, `start_time` should be set to `null`
-and `countdown_pause_time` to the number of seconds left to count down.
-The `countdown_pause_time` may change to indicate approximate delay.
+The typical use of `countdown_pause_time` is when a contest director wishes to pause the countdown to the start of a contest.  For example, this may occur because of technical
+issues or to make an announcement.  When the contest countdown is paused, the value of `countdown_pause_time` should be set to the expected time remaining before the start of the contest after the pause is lifted.
+In this case, the `start_time` must be set to `null`, thereby, setting an undefined `start_time` (the `start_time` is undefined since
```

Comment:
Clarify that we're referring to the `countdown_pause_time` attribute, not to the example use case in the previous sentences:
```suggestion
If `countdown_pause_time` is set, then the `start_time` must be set to `null`, thereby, setting an undefined `start_time` (the `start_time` is undefined since
```